contracts/lib/Structs.sol

```diff
@@ -18,7 +18,7 @@ library Structs {
         address tokenAddress;
         uint32 id;
         uint32 age;
-        uint32 epochLastUnstakedOrFirstStaked;
+        uint32 epochStakedOrLastPenalized;
```

Comment:
Hey, should we add 'first' before 'staked' here, becuase last time it was like this only, and then we added firstStaked as we thought thats proper name

Reply:
this pr : https://github.com/razor-network/contracts/pull/331

Reply:
Fixed https://github.com/razor-network/contracts/pull/443/commits/994f7726590db47175c2497b2b648bcebd1feae4